jck/jtrunner/JavaTestRunner.java

```diff
@@ -892,7 +893,8 @@ public static boolean jckConfigurationForMultijvmWithNoAgent() throws Exception
 			// Get any additional jvm options for specific tests.
 			extraJvmOptions += getTestSpecificJvmOptions(jckVersion, tests);
 
-			extraJvmOptions += suppressOutOfMemoryDumpOptions;
+			extraJvmOptions += suppressOutOfMemoryDumpOptions + " " + jvmOpts;
+			
 
 			// Add the JVM options supplied by the user plus those added in this method to the jtb file option.
 			fileContent += "set jck.env.devtools.refExecute.otherOpts \" " + extraJvmOptions + " \"" + ";\n";	
```

Comment:
This PR is not quite right. The EXTRA_OPTIONS is not added at the end of all other options.

I would suggest the following:
- revert the changes in the PR except line 494
- line 172
```
jvmOpts = System.getProperty("other.opts").trim() + " " + System.getProperty("jvm.options");
```
- in line 898
```
fileContent += "set jck.env.devtools.refExecute.otherOpts \" " + extraJvmOptions + jvmOpts + " \"" + ";\n";
```